src/main.rs

```diff
@@ -1427,6 +1427,13 @@ async fn hgive<'a>(slash_command: LenientForm<SlashCommand>) -> Json<Value> {
         Some(s) => s.as_str().to_string(),
         None => return res("Couldn't parse receiver?"),
     };
+    if &receiver == &slash_command.user_id {
```

Comment:
actually I think technically those `&` aren't necessary?

Reply:
I wanted to avoid memory errors thrown by the compiler, and it works just fine